libs/community/langchain_community/chat_models/yandex.py

```diff
@@ -120,11 +121,45 @@ async def _agenerate(
         message = AIMessage(content=text)
         return ChatResult(generations=[ChatGeneration(message=message)])
 
-
-def _make_request(
-    self: ChatYandexGPT,
-    messages: List[BaseMessage],
-) -> str:
+    def _stream(
+        self,
+        messages: List[BaseMessage],
+        stop: Optional[List[str]] = None,
+        run_manager: Optional[CallbackManagerForLLMRun] = None,
+        **kwargs: Any,
+    ) -> Iterator[ChatGenerationChunk]:
+        stream_resp = completion_with_retry(self, messages=messages, stream=True)
+        current_text = ""
+        for data in stream_resp:
+            delta = data[len(current_text) :]
```

Comment:
You can't do this, because the common prefix of the generations may change during streaming from yandex gpt :(